www/~/%username/routes/associate.json.spt

```diff
@@ -40,7 +40,13 @@ if change:
     elif network == 'paypal':
         if address not in participant.get_verified_email_addresses():
             raise Response(400, _("Only verified email addresses allowed."))
-        ExchangeRoute.insert(participant, network, address)
+        try:
+            ExchangeRoute.insert(participant, network, address)
+        except IntegrityError as e:
+            if e.pgcode != errorcodes.UNIQUE_VIOLATION:
+                raise
+            existing = ExchangeRoute.from_address(participant, network, address)
```

Comment:
Checking before insertion makes us prone to race conditions (two people check, both find no route, both try to write, one fails). 

I'm changing `existing` to `existing_route`